homepage/templates/homepage/base.html

```diff
@@ -35,24 +40,27 @@ <h1>Girls' Computer Club</h1>
 				</div>
 			</div>
 		</div>
-		<div class="footer">
-			<h2>Meet our board!</h2>
-			<div class="members-container">
-				<div class="member-info">
-					<img src="{% static 'images/logo.png' %}" width="100px">
+        <div class=footer>
+        <h2>Meet our board!</h2>
+		<div class="row">
+            <div class="col">
+            <img src="{% static 'images/logo.png' %}" width="100px">
 					<h3>Magda</h3>
+                    <p class="title">Founder</p>
 					<p>Lorem ipsum lipsum</p>
-				</div>
-				<div class="member-info">
-					<img src="{% static 'images/logo.png' %}" width="100px">
+        </div>
```

Comment:
Tässä sisennus muuttui vähän hassusti, jos voit vielä kurkata että parit on aina yhtä sisennetty:
```
<div>
    ...
</div>
```
Ja että ne sisennykset menee aina yhden askeleen (tabin) sisäänpäin kun mennään sisään vaiks diveihin tai linkkeihin!